src/compiler/crystal/command/format.cr

```diff
@@ -17,14 +17,15 @@ class Crystal::Command
     @format = "text"
     excludes = ["lib"] of String
     includes = [] of String
-    check = nil
+    check = false
```

Comment:
Maybe make this an instance var to avoid passing it around?

Reply:
`format` is `Crystal::Command`'s method, and `Crystal::Command` is extended by any files, so instance var conflicts easily. Other command methods under `src/compiler/crystal/command` doesn't use instance var also.

Reply:
Then maybe make it `@formatter_check`? It would simplify the code a lot.

Reply:
This pull request is for bug fix, and I have more ideas for refactoring `Crystal::Command#format` around. I'd like to work after merged this PR.